CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## 3.0.3-beta - 2022-11-29
+## 3.1.1-beta - 2023-01-31
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Maybe you could move the line 68 (`- YLP-2068 Update minor dependencies`) from `3.1.0` to `3.1.1` given that it's not included in the version to release